source/javascripts/core/models/BitriseYmlService.ts

```diff
@@ -259,7 +332,23 @@ function deleteWorkflowFromStages(workflowId: string, stages: Stages = {}): Stag
   });
 }
 
-function deleteWorkflowFromPipelines(workflowId: string, pipelines: Pipelines = {}, stages: Stages = {}): Pipelines {
+function renameWorkflowInPipelines(workflowId: string, newWorkflowId: string, pipelines: PipelinesYml): PipelinesYml {
+  return mapValues(pipelines, (pipeline) => {
```

Comment:
What's the use of `mapValues` here?

Reply:
Same as above... it modifies the original pipelines object.